src/components/dataset/schemaCreation/DatasetSchemaCreationView.test.tsx

```diff
@@ -0,0 +1,1021 @@
+import { mount } from '@cypress/react';
+import { Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { ThemeProvider } from '@mui/styles';
+import createMockStore from 'redux-mock-store';
+import React from 'react';
+import _ from 'lodash';
+import { initialUserState } from 'reducers/user';
+import DatasetSchemaCreationView from './DatasetSchemaCreationView';
+import globalTheme from '../../../modules/theme';
+import history from '../../../modules/hist';
+
+const initialState = {
+  searchString: '',
+  profiles: {
+    profiles: [{ profileName: 'default profile' }, { profileName: 'second profile' }],
+  },
+  user: _.cloneDeep(initialUserState),
+};
+
+// const fillValidInfoFields = () => {
```

Comment:
This file seems to be largely commented out code. Could we remove that? Do you still need it for some reason?